utils/script.sh

```diff
@@ -12,35 +27,25 @@ API_NAME="$2"
 DESTINATION_DIR="$3"
 
 # Define constants
-OPENAPI_GENERATOR_JAR=~/bin/openapitools/openapi-generator-cli.jar
-OPENAPI_URL_SERVICEINFO="https://raw.githubusercontent.com/ga4gh-discovery/ga4gh-service-info/develop/service-info.yaml"
-OPENAPI_URL_TES="https://raw.githubusercontent.com/ga4gh/task-execution-schemas/develop/openapi/task_execution_service.openapi.yaml"
-OPENAPI_GENERATOR_URL="https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/7.7.0/openapi-generator-cli-7.7.0.jar"
-SCRIPT_DIR="$(pwd)"
-SED_RULE="s/^use\s\+crate::models\s*;/#![allow(unused_imports)]\n#![allow(clippy::empty_docs)]\nuse crate::$API_NAME::models;/"
+SED_RULE="s/^use\s\+crate::models\s*;/#![allow(unused_imports)]\n#![allow(clippy::empty_docs)]\nuse crate::clients::$API_NAME::models;/"
 TEMP_OUTPUT_DIR=$(mktemp -d)  # Define the temporary output directory for the OpenAPI generator
 
 # Exit immediately if a command exits with a non-zero status.
 set -euo pipefail
 
-# Function to ensure the OpenAPI Generator JAR file is set up
-ensure_openapi_generator() {
-    mkdir -p ~/bin/openapitools
-    if [ ! -f "$OPENAPI_GENERATOR_JAR" ]; then
-        curl -L "$OPENAPI_GENERATOR_URL" -o "$OPENAPI_GENERATOR_JAR"
-        echo "d41d8cd98f00b204e9800998ecf8427e  $OPENAPI_GENERATOR_JAR" | sha256sum -c -
-    fi
-}
-
-# Call the function to ensure the OpenAPI Generator JAR file is set up
-ensure_openapi_generator
-
 generate_openapi_models() {
     # Remove the temporary directory at the end of the script
     trap 'rm -rf "$TEMP_OUTPUT_DIR"' EXIT
     
+    mkdir -p ~/bin/openapitools
+    curl https://raw.githubusercontent.com/OpenAPITools/openapi-generator/master/bin/utils/openapi-generator-cli.sh > ~/bin/openapitools/openapi-generator-cli
+    chmod u+x ~/bin/openapitools/openapi-generator-cli
+    export PATH=$PATH:~/bin/openapitools/
```

Comment:
I still think this is unnecessary and should rather go to the docs.

Reply:
Removed, and moved to documentation.